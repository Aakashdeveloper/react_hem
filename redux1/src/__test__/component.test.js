import React from 'react';
import TopicListing from '../component/topicListing';
import { create} from 'react-test-renderer';

describe('testing Component snapshot', () => {
    test('TopicListing', () => {
        let tree = create(<TopicListing/>)
        expect(tree.toJSON()).toMatchSnapshot();
    })
    test('TestHam component',() => {
        let tree = create(<TopicListing/>)
        expect(tree.toJSON()).toMatchSnapshot();
    })
})