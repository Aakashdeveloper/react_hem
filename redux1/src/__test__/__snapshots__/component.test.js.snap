// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Component snapshot TestHam component 1`] = `
<div
  className="panel panel-primary"
>
  <div
    className="panel-heading"
  >
    <h3>
      Topics app
    </h3>
  </div>
</div>
`;

exports[`testing Component snapshot TopicListing 1`] = `
<div
  className="panel panel-primary"
>
  <div
    className="panel-heading"
  >
    <h3>
      Topics app
    </h3>
  </div>
</div>
`;
